stockflux-securities-master/src/components/Message.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import "./Message.css";
+
+const Message = ({ message, type }) => {
```

Comment:
Good choice to rename this component and make it re-usable based on its type since an error message and success message component would be virtually identical

Reply:
Cheers